indico/migrations/versions/20240501_1431_67e92eeca34f_add_hide_registration_form.py

```diff
@@ -0,0 +1,31 @@
+"""Add hide registration form
+
+Revision ID: 67e92eeca34f
+Revises: 4e32f4d5ebe4
+Create Date: 2024-05-01 14:31:21.090459
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+
+# revision identifiers, used by Alembic.
+revision = '67e92eeca34f'
+down_revision = '4e32f4d5ebe4'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.add_column('forms', sa.Column('is_hidden', sa.Boolean(), nullable=False,
+                  server_default='false'), schema='event_registration')
+    op.alter_column('forms', 'is_hidden', server_default=None,
+                    schema='event_registration')
+    op.add_column('forms', sa.Column('uuid', postgresql.UUID(), nullable=True),
+                  schema='event_registration')
```

Comment:
I don't think this should be nullable, you can use this `server_default` to generate a new UUID for every form during the upgrade:

```py
            server_default=sa.text('''
                uuid_in(overlay(overlay(md5(random()::text || ':' || clock_timestamp()::text) placing '4' from 13)
                placing to_hex(floor(random()*(11-8+1) + 8)::int)::text from 17)::cstring)''')
```

Reply:
I did that on purpose to save data/operation as most of time regform will not be hidden,
but can set by default uuid for all if you want that.

Reply:
I think it's cleaner to always have it - the storage isn't significant for this